### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by RichardLitt**
Very minor.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Username to fetch repos from.
 
 #### options
 
-##### token
+##### `token`
```
Comment (kevva) [inline readme.md]: Oh, didn't notice in your previous PR. Can you remove the ticks around `endpoint` instead?

File: readme.md [hunk 2]
```diff
@@ -48,7 +48,7 @@ If you don't have a token you can generate a new one [here](https://github.com/s
 
 ##### `endpoint`
 
-Type: `string`
+Type: `string`
```
Comment (kevva) [inline readme.md]: Use `<br>` instead.

**Review by kevva — CHANGES_REQUESTED** [review]

**Comment by RichardLitt** [comment]
Fixed!

**Comment by kevva** [comment]
Thanks again!

**Pull request merged**